### PR TITLE
Replace deprecated SafeMarkup::checkPlain by Html::escape

### PR DESCRIPTION
**Pull request by GoZOo**
SafeMarkup::checkPlain is deprecated and should be replaced by Html::escape like suggested in Change records https://www.drupal.org/node/2549395

### CODE REVIEW

File: src/Form/ProfileTypeForm.php [hunk 1]
```diff
@@ -28,7 +28,7 @@ function form(array $form, FormStateInterface $form_state) {
     $type = $this->entity;
 
     if ($this->operation == 'add') {
-      $form['#title'] = SafeMarkup::checkPlain($this->t('Add profile type'));
+      $form['#title'] = Html::escape($this->t('Add profile type'));
```
Comment (bojanz) [inline src/Form/ProfileTypeForm.php]: This is a fixed string, so you can see there's nothing to escape :)

**Comment by bojanz** [comment]
Hm, these should actually be fine with no escaping, since Twig escapes them on its own end.
Can you test by removing the escaping, and seeing if any html in those variables is properly escaped?

**Comment by GoZOo** [comment]
If we use `Html::escape()`, result is wrong : `&lt;b&gt;Profil&#039;e&lt;script&gt;&lt;/b&gt;`

Html in this variables are not escaped at this state, but it will be with Twig. So no needs to use `Html::escape()`.

**Comment by mglaman** [comment]
The two items not wrapped in ````Html::escape()```` are covered via [Twig autoescape](https://www.drupal.org/node/2296163). Needs rebase, though.

File: src/Form/DeleteMultiple.php [hunk 2]
```diff
@@ -105,7 +105,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form['profiles'] = [
       '#theme' => 'item_list',
       '#items' => array_map(function ($profile) {
-        return SafeMarkup::checkPlain($profile->label());
+        return Html::escape($profile->label());
```
Comment (bojanz) [inline src/Form/DeleteMultiple.php]: node module's DeleteMultiple doesn't use Html::escape here at all.

**Comment by bojanz** [comment]
Fixed that and merged as a31b2915ddecdb04bb9243ee3c7077ed51e71ada.

**Pull request closed without merging**